compiler/src/main/kotlin/io/spine/protodata/CompilerEvents.kt

```diff
@@ -110,16 +110,20 @@ internal object CompilerEvents {
             .setPackageName(descriptor.`package`)
             .setSyntax(descriptor.syntax.toSyntaxVersion())
             .build()
-
         yield(
             FileEntered
                 .newBuilder()
                 .setFile(file)
                 .build()
         )
-
+        produceOptionEvents(descriptor.options) {
```

Comment:
In the declaration of the `produceOptionEvents` — as long as we are still at 100 symbol width —we should make
the `fun` declaration fit a single line.

(not able to leave a comment there, so writing it down here)